xml/System.Numerics/IShiftOperators`3.xml

```diff
@@ -101,11 +109,19 @@
         <Parameter Name="shiftAmount" Type="TOther" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <param name="shiftAmount">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="value">The value which is shifted right by <paramref name="shiftAmount" />.</param>
+        <param name="shiftAmount">The amount by which <paramref name="value" /> is shifted right.</param>
+        <summary>Shifts a value right by a given amount.</summary>
+        <returns>The result of shifting <paramref name="value" /> right by <paramref name="shiftAmount" />.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This operation is meant to perform n unsigned (otherwise known as a logical) right shift on all types.
```

Comment:
```suggestion
This operation is meant to perform an unsigned (otherwise known as a logical) right shift on all types.
```

Where else does this need to be updated?

Reply:
Not sure. If you find any additional instances of this typo in similar xmls, please submit a PR.